File: python/funcionarios/vendedor.py
from funcionario import Funcionario

class Vendedor(Funcionario):
    
    def __init__(self, nome, CPF, comissao):
        super().__init__(nome, CPF)
        self.comissao = comissao
        
    def calculaSalario(self):
        return Funcionario.salarioBase + self.comissao

File: java/funcionrios/Vendedor.java
public class Vendedor extends Funcionario{
    private double comissao;

    public Vendedor(String nome, String CPF, double comissao) {
        super(nome, CPF);
        this.comissao = comissao;
    }

    public double calculaSalario() {
        return this.getSalarioBase() + this.comissao;
    }
}
